com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/GraphicsToolsStandard.shader

```diff
@@ -40,7 +40,8 @@ Shader "Graphics Tools/Standard"
         [Toggle(_RIM_LIGHT)] _RimLight("Rim Light", Float) = 0.0
         _RimColor("Rim Color", Color) = (0.5, 0.5, 0.5, 1.0)
         _RimPower("Rim Power", Range(0.0, 8.0)) = 0.25
-        [Toggle(_VERTEX_COLORS)] _VertexColors("Vertex Colors", Float) = 0.0
+        [Toggle(_VERTEX_COLORS)] _VertexColors("Vertex Colors", Float) = 0
+        [Toggle(_VERTEX_BENTNORMALAO)] _VertexBentNormalAo("Vertex AO", Float) = 0
```

Comment:
This is more of an FYI, but we also have a Standard Canvas version: https://github.com/microsoft/MixedReality-GraphicsTools-Unity/blob/main/com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/GraphicsToolsStandardCanvas.shader

I was wondering if it needs AO support as well? But probably not? I don't think we would be baking AO in UIs?

Reply:
For the most part, UI is pretty sparse vertex-wise and I don't think would get the same benefit out of the box - though, having AO like shadows for UI had crossed my mind. Maybe if this graduates from Experimental? (I have some other ideas like adding mesh triangulation to the tool that could help here)